File: Kamil/SkryptyC#/Ksef_API_ASP.Net/Ksef_API_ASP.Net/Models/FakturaDTO.cs
using Ksef.Models;

namespace Ksef_API_ASP.Net.Models
{
    public class FakturaDTO
    {

        public required PodmiotDTO Sprzedawca{ get; set; } // sprzedawca
        public required PodmiotDTO Nabywca { get; set; } // nabywca
        public required string KodWaluty { get; set; }
        public required DateTime DataWyslania { get; set; } 
        public required string NrFaktury { get; set; } 
        public required DateTime DataOd { get; set; } 
        public required DateTime DataDo { get; set; }
        public  decimal KwatoaNetto { get; set; } 
        public  decimal KwotaPodatku { get; set; } 
        public  decimal KwotaPodatkuPLN { get; set; } 
        public  decimal KwotaNaloznosci { get; set; } 
        public  List<FaWierszDTO> FaWiersze { get; set; }
       
    }
}

File: Kamil/SkryptyC#/Ksef_API_ASP.Net/frontend/src/Models/API/FakturaDTO.ts
import type { PodmiotDTO } from "./PodmiotDTO"; 
import type { FaWierszDTO } from "./FaWierszDTO"; 

export interface FakturaDTO{
	sprzedawca: PodmiotDTO,
	nabywca: PodmiotDTO,
	kodWaluty: string,
	dataWyslania: Date,
	nrFaktury: string,
	dataOd: Date,
	dataDo: Date,
	kwatoaNetto: number,
	kwotaPodatku: number,
	kwotaPodatkuPLN: number,
	kwotaNaloznosci: number,
	faWiersze?: FaWierszDTO[],
	dodatkoweInformacje?: object,
}